db/data_migration/20150424094040_republish_coming_soon.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(Edition.scheduled).perform
```

Comment:
Actually, I don't think this will work at all. The PublishingApiRepublisher calls `PublishingApi::publish_async`, which raises an error for non-live editions. You'd need to iterate through `Edition.scheduled` and explicitly call `PublishingApi::schedule_async` which will do the right thing.